pointers/pointers.c

```diff
@@ -6,9 +6,11 @@
     address it's referring to) or the value at the address it's 
```

Comment:
pointers are the most difficult part of C, your code shows you've been studying them